### PR TITLE
Add login modules for Zabbix & Chef WebUI

### PR DESCRIPTION
**Pull request by hdm**
Stock Zabbix 2.4.1 appliance (LiveCD ISO): 

```
$ rm -f ~/.msf4/config && ./msfconsole -q
msf > use auxiliary/scanner/http/zabbix_login
msf auxiliary(zabbix_login) > set RHOSTS 192.168.254.123
msf auxiliary(zabbix_login) > run
[*] 192.168.254.123:80 ZABBIX - Found Zabbix version 2.4.1
[+] 192.168.254.123:80 ZABBIX - Note: This Zabbix instance has Guest mode enabled
[+] 192.168.254.123:80 ZABBIX - Success: 'Admin:zabbix'
[*] Scanned 1 of 1 hosts (100% complete)
[*] Auxiliary module execution completed
```

After disabling Guest mode:

```
[*] 192.168.254.123:80 ZABBIX - Found Zabbix version 2.4.1
[*] 192.168.254.123:80 ZABBIX - Zabbix has disabled Guest mode
[+] 192.168.254.123:80 ZABBIX - Success: 'Admin:zabbix'
```

After changing the password:

```
[*] 192.168.254.123:80 ZABBIX - Found Zabbix version 2.4.1
[*] 192.168.254.123:80 ZABBIX - Zabbix has disabled Guest mode
```

Stock Chef WebUI installation (via official curl|sh):

```
$ rm -f ~/.msf4/config && ./msfconsole -q
msf > use auxiliary/scanner/http/chef_webui_login
msf auxiliary(chef_webui_login) > set RHOSTS 192.168.254.124
msf auxiliary(chef_webui_login) > run
[*] 192.168.254.124:443 CHEF_WEBUI - Found Chef Web UI application at /
[+] 192.168.254.124:443 CHEF_WEBUI - Success: 'admin:p@ssw0rd1'
[*] Scanned 1 of 1 hosts (100% complete)
[*] Auxiliary module execution completed
```

After changing the password:

```
[*] 192.168.254.124:443 CHEF_WEBUI - Found Chef Web UI application at /
```

### CODE REVIEW

**Comment by todb-r7** [comment]
Oooh there's specs too!

File: modules/auxiliary/scanner/http/chef_webui_login.rb [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        )
+      end
+    end
+  end
+
+
+  #
+  # main
+  #
+  def run_host(ip)
```
Comment (dmaloney-r7) [inline modules/auxiliary/scanner/http/chef_webui_login.rb]: super nitpicky, but can you please move run_host to the top and then lexical sort the other methods? It's much easier to read through these if the entry-point is first and the rest are in lexical ordering.

File: modules/auxiliary/scanner/http/zabbix_login.rb [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        )
+      end
+    end
+  end
+
+
+  #
+  # main
+  #
+  def run_host(ip)
```
Comment (dmaloney-r7) [inline modules/auxiliary/scanner/http/zabbix_login.rb]: please move entrypoint method to the top and lexical sort the other methods

**Comment by dmaloney-r7** [comment]
everything else looks good here,. i'll do the reordering locally before I merge at this point. Specs all good. code looks fine, and everything even has YARD docs. hooray!

File: modules/auxiliary/scanner/http/zabbix_login.rb (hunk 2 above)
Comment (hmoore-r7) [inline modules/auxiliary/scanner/http/zabbix_login.rb]: I copied the syntax from an existing module (glassfish from @jlee-r7).

**Comment by hmoore-r7** [comment]
Thanks!

**Pull request merged**